File: Python-Assignment/Chapter-Three/Question1.java
import java.util.Scanner;

public class Question1{
   public static void main(String []args){

	Scanner input = new Scanner(System.in);

	int passes = 0;
	int failures = 0;

	for(int students = 1; students <= 10; students++){
		System.out.print("Enter a score (1 = pass and 2 = fail): ");
		int score = input.nextInt();

		if (score == 1)
			passes = passes + 1;
		else if(score == 2)
			failures = failures + 1;
		else
			continue; 
	}

		System.out.println("The number of student that passed: " + passes);
		System.out.println("The number of student that failed: " + failures);

	if (passes >= 8)
		System.out.print("Great Job!!! Bonus to the instructor");
   }

}

File: Python-Assignment/Chapter-Three/question1.py
passes = 0
failures = 0

for student in range(10):
	score = int(input("Enter a score (1 = pass and 2 = fail): "))

	if score == 1:
		passes = passes + 1
	elif score == 2:
		failures = failures + 1
	else:
		continue

print("The number of student that passed: ", passes)
print("The number of students that failed: ", failures)

if passes >= 8:
	print("Great Job!!! Bonus to the instructor")